api/types/storage_plans.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+type StoragePlan struct {
+	ID                  string `json:"id" header:"ID"`
+	Name                string `json:"name" header:"NAME"`
+	MinSize             int    `json:"min_size,omitempty" header:"MIN_SIZE"`
+	MaxSize             int    `json:"max_size,omitempty" header:"MAX_SIZE"`
+	CloudProviderID     string `json:"cloud_provider_id,omitempty" header:"CLOUD_PROVIDER_ID"`
+	CloudProviderName   string `json:"cloud_provider_name" header:"CLOUD_PROVIDER_NAME"`
```

Comment:
CloudProviderName is not feeded by the API representation, maybe it should not have a json struct tag

Reply:
Done, at this point; but in addition, I take note about this. In near dates I’m going to review structures, since this is something to review along all calculated properties